api/prisma/seed-helpers/alternate-contact-factory.ts

```diff
@@ -0,0 +1,43 @@
+import { Prisma } from '@prisma/client';
+import { randomInt } from 'crypto';
+import { randomAdjective, randomNoun } from './word-generator';
+import { AlternateContactRelationship } from '../../src/enums/applications/alternate-contact-relationship-enum';
+import { addressFactory } from './address-factory';
+
+export const alternateContactFactory =
+  (): Prisma.AlternateContactCreateWithoutApplicationsInput => {
+    const relationshipKeys = Object.values(AlternateContactRelationship);
+    const firstName = randomNoun();
+    const lastName = randomNoun();
+    const relationshipType =
+      relationshipKeys[randomInt(relationshipKeys.length)];
+
+    if (relationshipType === AlternateContactRelationship.noContact) {
+      return {
+        firstName: undefined,
+        lastName: undefined,
+        type: relationshipType,
+        otherType: undefined,
+        phoneNumber: undefined,
+        emailAddress: undefined,
+        agency: undefined,
+        address: { create: addressFactory() },
```

Comment:
This is technically inaccurate but we're sending a strange empty-but-with-id address on alternate contact through the frontend on actual application submission and that felt like a bigger problem